### DISCUSSION TITLE
download api for file

### DISCUSSION THREAD

**Discussion by nithin-techforce** [directus/directus#4618] [Archive]
I want to fetch a file data and download it using REST api ,in documentation i see get api which gives the data about a file but not actual contents of it can anyone share the process how to get the file contents

**Comment by rijkvanzanten** [comment]
See https://docs.directus.io/reference/api/assets/

**Reply by alexvdvalk** [reply]
Updated URL is https://docs.directus.io/reference/files.html#downloading-a-file
